elastic/kibana
# [Synthetics] report error when synthetics agent sends an invalid location

## Summary

Resolves https://github.com/elastic/kibana/issues/153586

Reports and error when an invalid location or private location is sent from the synthetics agent.

### Testing
1. Create a test project directory via synthetics by using `npx @elastic/synthetics init`, or use an existing project
2. Create a private location in the Synthetics UI
3. Attempt to push monitors from your project with a valid public and private location. Confirm your push is successful.
4. Change the private location to an invalid private location. Attempt to repush. Confirm that you received an error.
5. Change the private location back to a valid private location. Change the public location to an invalid public location. Attempt to repush. Confirm you receive an error.
